components/concourse-task-toolbox/Dockerfile

```diff
@@ -27,9 +27,12 @@ RUN apk add --update \
 	less \
 	python \
 	py-pip \
+	python3 \
```

Comment:
Just a thought, would we need [python3.7](https://packages.ubuntu.com/disco/python3.7)?